app/ui/colourLibrary.ts

```diff
@@ -0,0 +1,6 @@
+export const darkGrey = "#535353";
+export const darkBlueBackground = "#090043";
+export const black = "#000000";
+export const white = "#ffffff";
+export const greyInactiveButton = "#B5A8A8";
+export const redActiveButton = "#E86868";
```

Comment:
there are probably a bunch of places that are still using the codes/strings for colours instead of these constants, but we can just find them as we go along